cmake/configure/configure_2_trilinos.cmake

```diff
@@ -38,17 +38,17 @@ MACRO(FEATURE_TRILINOS_FIND_EXTERNAL var)
     # Check whether all required modules of trilinos are installed:
     #
     MESSAGE(STATUS
-      "Check whether the found trilinos package contains all required modules:"
+      "Checking whether the found trilinos package contains all required modules:"
```

Comment:
This isn't supposed to be a command ("check!") but a status update about what cmake is currently doing.